recipes/panel-chemistry/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "panel-chemistry" %}
+{% set version = "0.0.11" %}
+{% set sha = "5d780a29e0ae31bce0b03cca6906c29abb9c27152c8cb5de9bb5489308179f4c" %}
+
+
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/panel-chemistry-{{ version }}.tar.gz
+  sha256: {{ sha }}
```

Comment:
Please put the sha directly here

Reply:
I will. It was initially there. I changed it because I forgot to update the sha when updating the package version and I figured it would be easier to remember to do if it was a the top together.

Reply:
Done @xhochy